src/python/PSetTweaks/WMTweak.py

```diff
@@ -388,7 +389,9 @@ def makeTaskTweak(stepSection, result):
     if hasattr(stepSection, "application"):
         if hasattr(stepSection.application, "configuration"):
             if hasattr(stepSection.application.configuration, "pickledarguments"):
-                args = pickle.loads(stepSection.application.configuration.pickledarguments)
+                pklArgs = encodeUnicodeToBytesConditional(stepSection.application.configuration.pickledarguments,
+                                                          condition=PY3)
+                args = pickle.loads(pklArgs)
```

Comment:
I like that you are trying to fix only what breaks, but `pickle.loads()` expects the data to be of type bytes both in py2 and py3, so we could avoid using the conditional here. but do not worry, we should not have problem!

Reply:
Yes, I wanted to be as conservative as possible. At least in central production, I can confirm that that attribute comes as bytes (since we migrated services to Py3).
Okay, I'm not going to update this patch then (otherwise I have to re-run my tests). But we better leave it as unresolved for future reference. Thanks!